apps/api/src/app/events/usecases/process-subscriber/process-subscriber.command.ts

```diff
@@ -16,7 +11,7 @@ export class ProcessSubscriberCommand extends EnvironmentWithUserCommand {
   payload: any; // eslint-disable-line @typescript-eslint/no-explicit-any
 
   @IsDefined()
-  to: ISubscribersDefine; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
I've noticed a few fields where this is not needed anymore because the type is defined